client/my-sites/backup/backup-contents-page/file-browser/file-preview.tsx

```diff
@@ -91,7 +94,7 @@ const FilePreview: FunctionComponent< FilePreviewProps > = ( { item, siteId } )
 		return content;
 	};
 
-	const isLoading = isTextContent ? ! fileContent : isInitialLoading;
+	const isLoading = isTextContent ? ! fileContent && ! isError : isInitialLoading;
```

Comment:
In case of an error loading a file preview, we are just hiding the loading placeholder. Not sure if adding an error notice for file previews would be a good idea, as technically, they don't have a way to re-trigger it manually.